src/main/java/com/minecolonies/coremod/entity/ai/citizen/lumberjack/Tree.java

```diff
@@ -391,6 +454,21 @@ public BlockPos peekNextLog()
     }
 
     /**
+     * Looks up the next leaf block.
+     *
+     * @return the position.
+     */
+    public BlockPos peekNextLeaf()
+    {
+        return leaves.peekLast();
+    }
+
+    public boolean hasleaves()
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)